integration/client-cli/client-registration-cli/src/main/java/org/keycloak/client/registration/cli/util/HttpUtil.java

```diff
@@ -185,4 +189,22 @@ public static void setTruststore(File file, String password) throws CertificateE
                 .build();
         sslsf = new SSLConnectionSocketFactory(theContext);
     }
+
+    public static void setSkipCertificateValidation() {
```

Comment:
If my comments to the other `HttpUtil` class are valid, they also apply here.

Reply:
I think that should be fine. Let's continue the discussion on the comment above.